ui/components/playground/PlaygroundContainer.js

```diff
@@ -1,16 +1,30 @@
 import React from "react";
 import { Provider } from "react-redux";
 import { Playground, store } from "graphql-playground-react";
+import { Query } from "react-apollo";
+import { FormControl } from "patternfly-react";
+
+import GetGraphQLEndpoint from "../../graphql/GetGraphQLEndpoint.graphql";
 
 const settings = {
     "editor.theme": "light",
     "editor.cursorShape": "line"
 };
 
 const PlaygroundContainer = () => (
-    <Provider store={store}>
-        <Playground endpoint="http://localhost:8000/graphql" settings={settings} />
-    </Provider>
+    <Query query={GetGraphQLEndpoint}>
+        {({ loading, error, data }) => {
+            if (loading || typeof error !== "undefined") {
+                return <FormControl.Static>Loading graphql playground...</FormControl.Static>;
+            }
+            const { getGraphQLEndpoint } = data;
+            return (
+                <Provider store={store}>
+                    <Playground endpoint={getGraphQLEndpoint} settings={settings} />
```

Comment:
😍

Reply:
@darahayes Why we need to do this now? Are we trying to build project functionality? 
This has too big scope already.  PR and work is not designed to work with the projects abstractions and as you mentioned it will be trivial to implement, however let's not add more to this work and focus on the targets of this sprint. 

TL;DR - with this changes projects abstraction will work, but let's not add it here.
I was hoping to reduce scope of this changes and leave some obvious places where we can hook on later.

Reply:
Actually if that is small change then it's ok. I would to this exactly as suggested. 
This PR seems awesome.